erpnext/www/book-appointment/index.js

```diff
@@ -0,0 +1,214 @@
+frappe.ready(async () => {
+    initialise_select_date();
+})
+
+window.holiday_list = [];
+
+async function initialise_select_date() {
+    navigate_to_page(1);
+    await get_global_variables();
```

Comment:
Why await here? The promise is not stored anywhere.. The program will just go through the execution. There is no way to ensure that the API calls returned the values